### ISSUE TITLE
Suggestion: Balancing changes

### ISSUE THREAD

**Issue by stormdirus2** [Ladysnake/Requiem#229]
**1. Add 15-30s cooldowns for the healing items for each respective hostile mob (possibly use some status effect to control this). Moderately to heavily reduce drops for said items without a humanity enchantment.**

**2. Add hunger back to hostile mobs, there would only be two options to refill it:**
    -Damaging villagers, players, piglins, witches, illagers, evokers, or iron/snow golems. (Basically all humanoids except endermen and undead)
    -Consuming the respective healing item for the possessed hostile mob.
Additionally, saturation would not be replenished by damage alone. It would require one of the healing items, or for one of the above mobs to be killed. The latter option replenishing only a little saturation.

**3. Apply glowing to possessable mobs near the player while they are dead. This is to decrease the boring moments where the player finds themselves having to scour around just to find a host.** 
    -Possibly intervene to summon a possessable mob in a close dark area when there are none within a certain vicinity.

**Comment by Pyrofab** [comment]
Thanks for the suggestions ! I think the first one is very reasonable, I see no reason not to add it. The second and third ones are a bit more involved, but they are interesting. We'll definitely have to think about a way to find possessable mobs more easily as a dissociated soul.

**Comment by stormdirus2** [comment]
@Pyrofab 
> Thanks for the suggestions ! I think the first one is very reasonable, I see no reason not to add it. 
>
Thanks! I'm glad you liked it. I noticed that I could instantly heal as fast as I could eat and realized the possible implications of it. Under the right circumstances it could make being undead _really_ advantageous, especially paired with the fact that 95% of mobs no longer attack you.

>The second and third ones are a bit more involved, but they are interesting.
>
Thanks! It's ok if the changes might be outside the scope of what you wanted in the mod or a bit invasive of a change to adapt without lots of thought.

>We'll definitely have to think about a way to find possessable mobs more easily as a dissociated soul.
>
Great! One of the things I liked about this mod is how it removed the need to backtrack long distances just to regain old gear and does so in an incredibly interesting way. The stuff on the roadmap is also going to add a lot of unique and fun game elements as well. The only caveat with how it tries to allow this is that it's occasionally quite hard to find a host. At one point I died in broad daylight from fall damage and it took quite a while to locate a host.


**Additional suggestion:**
I was thinking about a suggestion that was made earlier; it poked around with the idea of having mobs slowly reclaim their bodies. I was thinking of a similar suggestion, but less complex.

With the wandering spirit mode added, it is possible to be more aggressive with the balancing while letting people sustainably roleplay as undead via the wandering spirit.

**I think after a certain arbitrary amount of days in a hostile host, the attrition level should increase.**
This would reduce max health like normal and instigate a sense of urgency as the player's health slowly decreases each interval.

If this were implemented, I would suggest that a gamerule be added to control the amount of days between each increase.

Also, in light of the road map, the attrition level would not increase while in mobs who have had their soul removed or mobs like cured piglins. This would provide further incentive removing a mob's soul as it would be a safer host if the player were to die.
